storm-webapp/src/main/java/org/apache/storm/daemon/logviewer/webapp/LogviewerResource.java

```diff
@@ -229,9 +230,12 @@ public Response listDumpFiles(@PathParam("topo-id") String topologyId, @PathPara
     @Path("/dumps/{topo-id}/{host-port}/{filename}")
     public Response downloadDumpFile(@PathParam("topo-id") String topologyId, @PathParam("host-port") String hostPort,
                                      @PathParam("filename") String fileName, @Context HttpServletRequest request) throws IOException {
+        InetAddress inetAddress = InetAddress.getLocalHost();
```

Comment:
Better to put these in the try-catch block since we increase `numDownloadDumpExceptions`  if any exception happens. There are a few places like this